src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -15,14 +16,22 @@
 MAX_TRACE_SIZE = 100
 NODESTORE_KEYS = ["timestamp", "start_timestamp"]
 DETAILED_NODESTORE_KEYS = ["environment", "release"]
+ERROR_COLUMNS = [
+    "id",
+    "project",
+    "event.type",
+    "timestamp",
+    "trace.span",
+    "transaction",
+]
 
 
 def find_event(items, function, default=None):
     return next(filter(function, items), default)
 
 
 def is_root(item):
-    return item["root"] == "1"
+    return item.get("root", "0") == "1"
```

Comment:
Did this so we could re-use `construct_span_map` for both errors & transactions, and errors will never be the "root" event